src/olympia/access/acl.py

```diff
@@ -71,9 +72,9 @@ def check_collection_ownership(request, collection, require_owner=False):
     if not request.user.is_authenticated():
         return False
 
-    if action_allowed(request, 'Admin', '%'):
+    if permissions.ADMIN.has_permission(request):
         return True
-    elif action_allowed(request, 'Collections', 'Edit'):
+    elif permissions.COLLECTIONS_EDIT.has_permission(request):
```

Comment:
making it the constants just tuples would still achieve the aims of consistently using permissions; decorator and DRF compatibility could be wrappers, used when needed.